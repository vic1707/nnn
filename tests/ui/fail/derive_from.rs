use nnn::nnn;

#[nnn(derive(From))]
struct FromFloat(f64);

fn main() {}
