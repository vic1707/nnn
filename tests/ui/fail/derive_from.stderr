error: Deriving `From` results in a possible bypass of the validators and sanitizers and is therefore forbidden.
 --> tests/ui/fail/derive_from.rs:3:14
  |
3 | #[nnn(derive(From))]
  |              ^^^^
